openmdao/core/system.py

```diff
@@ -2605,15 +2618,6 @@ def _solve_linear(self, vec_names, mode):
             list of names of the right-hand-side vectors.
         mode : str
             'fwd' or 'rev'.
-
```

Comment:
OH, the diff I'm seeing looked like a blank line got deleted.  I'll restore it.  I'm not sure why that would have been on my radar to delete.  Likely an error.